### ISSUE TITLE
AuditRetentionPeriod cannot be set on current online environment

### ISSUE THREAD

**Issue by dklinger** [seanmcne/OrgDbOrgSettings#77]
Hi,
we are not able to change the value of AuditRetentionPeriod. Neither updating, nor resetting or removing is possible.
Getting the following error message:

We've handled an error, for reference the error info is:
Name:Error
Message:There was an error updating your orgDBOrgSettings: Invalid Organization Setting passed in: AuditRetentionPeriod. Please check the datatype and pass in an appropriate value.
Description:Error Description Is Null
stack:Error: There was an error updating your orgDBOrgSettings: Invalid Organization Setting passed in: AuditRetentionPeriod. Please check the datatype and pass in an appropriate value. at Object.error (https://orangecrm-hog.crm4.dynamics.com//WebResources/mspfedyn_/orgdborgsettings/orgdborgsettings.html:191:40) at j (https://orangecrm-hog.crm4.dynamics.com/_static/_common/scripts/jquery-2.1.1.min.js?ver=-277899311:1417:29) at Object.fireWith [as rejectWith] (https://orangecrm-hog.crm4.dynamics.com/_static/_common/scripts/jquery-2.1.1.min.js?ver=-277899311:1467:104) at x (https://orangecrm-hog.crm4.dynamics.com/_static/_common/scripts/jquery-2.1.1.min.js?ver=-277899311:3385:605) at XMLHttpRequest. (https://orangecrm-hog.crm4.dynamics.com/_static/_common/scripts/jquery-2.1.1.min.js?ver=-277899311:3555:96)

Happy for any info you can give.

**Comment by seanmcne** [comment]
Thanks for reporting this!  It does appear the setting has been deprecated/removed in favor of a different name (due to a change in default values to accommodate how the platform is storing audit data now).

**Comment by dklinger** [comment]
Thank you for the quick response.
I'm note sure what your answer means. https://support.microsoft.com/de-de/topic/orgdborgsettings-tool-for-microsoft-dynamics-crm-20a10f46-2a24-a156-7144-365d49b842ba is still stating the old name.
Will you release an updated version of OrgDbOrgSettings or is this name change currently in progress at MS side?

**Comment by seanmcne** [comment]
Yes, the KB is incorrect or doesn't mention the fact this setting is no longer valid.  As for the udpated/different setting I will look into that.  Thx!
